Source/Core/DolphinQt/RenderWidget.cpp

```diff
@@ -254,6 +254,10 @@ void RenderWidget::SetCursorLocked(bool locked, bool follow_aspect_ratio)
   if (locked)
   {
 #ifdef _WIN32
+    // This will prevent the mouse from interacting with the task bar
+    // while in windowed / borderless fullscreen.
+    SetWindowPos((HWND) winId(), HWND_TOPMOST, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
```

Comment:
You should probably do this within the `if (ClipCursor(&rect))` condition, as if ClipCursor fails, you don't want to set the window to the topmost. Even if ClipCursor might never fail, it would still be better I'd say.